docs/source/examples/vision/checks/performance/source/plot_robustness_report.py

```diff
@@ -6,41 +6,41 @@
 
 **Structure:**
 
-* `Why augmentations on images should be used? <#why-augmentations-on-images-should-be-used>`__
-* `How does model robustness on images is checked? <#how-does-model-robustness-on-images-is-checked>`__
+* `How Does the RobustnessReport Check Work? <#how-does-the-robustness-report-check-work>`__
+* `What Are Image Augmentations? <#what-are-image-augmentations>`__
 * `Check requirements <#check-requirements>`__
 * `Generate data and model <#generate-data-and-model>`__
 * `Run the check <#run-the-checkl>`__
 * `Define a condition <#define-a-condition>`__
 
-Why augmentations on images should be used?
+How Does the RobustnessReport Check Work?
+===============================================
+This check performs augmentations on images in the test dataset, and measures the change
```

Comment:
By default it would run also on train if run in a suite. Nothing in the check enforces running only on test.

Reply:
First of all, not my part of the text :P But... Well... that's bad.
Shouldn't it?